scripts/integration-contracts.sh

```diff
@@ -0,0 +1,22 @@
+account_key=0xe82294532bcfcd8e0763ee5cef194f36f00396be59b94fb418f5f8d83140d9a7
```

Comment:
Does this associate with any account? Should it be a secret?

Reply:
this is actually the first key from the first 10 local blockchain accounts, when u use a fixed mnemonic to seed the blockchain. its deterministic as long as u used a fixed phrase + it's from local accounts so i think should be fine